tests/test_utils.py

```diff
@@ -7,6 +7,8 @@
 import pytest
 
 import audeer
+from pandas._libs.tslibs.nattype import NaT
```

Comment:
I recommend to remove this important and use `pd.NaT` in the tests below.

Reply:
Okay, why the recommendation tho ?

Reply:
1. We already import `pandas`, so there is no need for an extra import
2. You are importing from a private module (`_libs`) here, which is not a good idea. They may change it in a future version, but `pd.NaT` will always be there.

Reply:
Okay understandable thanks, applied.